migration/v2.19.0-ck8s1-v2.19.0-ck8s2/upgrade-cluster.md

```diff
@@ -0,0 +1,56 @@
+# Upgrade v2.19.0-ck8s1 to v2.19.0-ck8s2
+
+1. Checkout the new release: `git checkout v2.19.0-ck8s2`
+
+1. Switch to the correct remote: `git submodule sync`
+
+1. Update the kubespray submodule: `git submodule update --init --recursive`
+
+1. This patch includes options to create OIDC kubeconfigs instead of regular kubeconfigs. If you want to switch to relying on using OIDC kubeconfigs, then follow "Switch to OIDC kubeconfig" below, otherwise follow "Keep using regular kubeconfigs" below.
+
+## Switch to OIDC kubeconfig
+
+1. Make the following changes to both `sc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and `wc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml`
```

Comment:
Can be more specific that sc should be connected to google and wc connected to wc. Also what the secret key is called containing the dex client secret.